### PR TITLE
Expose VM configuration in budget

### PR DESCRIPTION
**Pull request by malteo**

### CODE REVIEW

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetDTO.java [hunk 1]
```diff
@@ -72,6 +73,12 @@
 
     private IaasBilling iaasBilling;
 
+    private OSType operatingSystem;
+
+    private Integer rootDiskSpace;
+
+    private Integer dataDiskSpace;
```
Comment (gionn) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetDTO.java]: refactor in a new class to keep them together

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetDTO.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.liberologico.cloudesire.common.MathConfiguration;
 import com.liberologico.cloudesire.common.MathUtils;
+import com.liberologico.cloudesire.common.enums.OSType;
```
Comment (AndreaVecchi) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetDTO.java]: MINOR Remove this unused import 'com.liberologico.cloudesire.common.enums.OSType'. [rule](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by AndreaVecchi** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetVMConfigurationDTO.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
+package com.cloudesire.platform.apiclient.dto.model.dto;
+
+import com.liberologico.cloudesire.common.enums.OSType;
+import io.swagger.annotations.ApiModel;
+
+@ApiModel( description = "" )
```
Comment (gionn) [inline cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetVMConfigurationDTO.java]:
```suggestion
@ApiModel( description = "Which cloud resources the customer will get for this order" )
```

**Pull request merged**